.github/workflows/dusk_ci.yml

```diff
@@ -0,0 +1,46 @@
+on: [pull_request]
+
+name: Continuous integration
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
+
+  test_nightly:
+      name: Nightly tests
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+        - uses: actions-rs/toolchain@v1
+          with:
+            profile: minimal
+            toolchain: nightly
```

Comment:
As discussed let's remove the toolchain since we agreed on using rust-toolchain file.
The same should be done for bls repo as well.

Reply:
Fixed. Thanks.